src/main/resources/assets/app/scripts/views/job_detail_view.js

```diff
@@ -36,12 +36,10 @@ function($,
 
   'use strict';
 
```

Comment:
Very attentive. :)
asyncExecutorPath isn't used anywhere and is abandoned, so I decided to tidy up a little bit.